### ISSUE TITLE
x/build: support x/playground tests that need additional dependencies

### ISSUE THREAD

**Issue by dmitshur** [golang/go#66002]
Right now the x/playground repo is tested via the usual `go test ./...` as other x/ repos.  
As described in #24823, there are tests that need additional dependencies to run (Docker).  
This is the tracking issue for that.

CC @golang/tools-team.

**Comment by findleyr** [comment]
As described in #69403, while I think we *should* support Dockerized tests for some projects such as vscode-go or pkgsite, my intuition is that an the playground relies on Docker more than it really needs to (it's just running the go command, after all!). We should probably address this issue by removing Docker from the development and test flow in x/playground.

**Comment by dmitshur** [comment]
Agreed – if achieving x/playground's testing needs can be made to work without the Docker dependency, that path would be great. Recently @hyangah is exploring a similar direction for vscode-go's tests, tracked in https://github.com/golang/vscode-go/issues/3533.
